plugins/@grouparoo/hubspot/src/lib/export-contacts/exportRecord.ts

```diff
@@ -88,6 +97,9 @@ export const exportRecord: ExportRecordPluginMethod = async ({
       );
       const formattedDataFields = {};
       for (const key of Object.keys(payload)) {
+        if (companyMappingKey && key === companyMappingKey) {
```

Comment:
something funny her about `old` Company keys. Like the setting got switched from `domain` to `company_id`
Maybe it should skip anything that starts with `Company.` 
Then the old one would get sent to the contact.